docs/Framework.md

```diff
@@ -708,9 +708,9 @@ typedef NcUint32    NcOid; // Object id
 
 ### NcOrganizationId
 
-Unique 24-bit organization ID:
+Unique 24-bit organization id:
 
-- IEEE public Company ID (public CID) or
+- IEEE public Company id (public CID) or
 - IEEE Organizational Unique Identifier (OUI).
```

Comment:
I think the correct term is Organizationally Unique Identifier, see https://standards.ieee.org/faqs/regauth/.

Reply:
Good spot. Should we correct this term now @jonathan-r-thorpe since we already have the PR open?

Reply:
OK, I will update accordingly